src/components/FullWidthPromo/FullWidthPromo.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import { ImageElement } from 'Image';
+import Button from 'Button';
+import Icon from 'Icon';
+import Link from 'Link';
+import RichText from 'RichText';
+import TagList, { TagProps } from 'TagList/TagList';
+
+type FullWidthPromoProps = {
+  authors?: string[];
+  description?: string;
+  href: string;
+  imageAlt?: string;
+  imageCaption?: string;
+  imageCredit?: string;
+  imageLicence?: string;
+  imageSizes?: string;
+  imageSrc?: string;
+  imageSrcSet?: string;
+  linkText: string;
+  metaLabel?: string;
+  title: string;
+  topics: TagProps[];
+};
+
+export const FullWidthPromo = ({
+  authors,
+  description,
+  href,
+  imageAlt,
+  imageSizes,
+  imageSrc,
+  imageSrcSet,
+  linkText,
+  metaLabel,
+  title,
+  topics
+}: FullWidthPromoProps) => (
+  <div className="cc-fw-promo">
```

Comment:
What's the benefit of abbreviating the classname? 

I'm thinking that it would make it more difficult to track down the React component if, say, you were only looking at the rendered HTML and wanted to find the relevant component in the codebase.

Reply:
Simply for brevity. TBH I'm not entirely sold on the naming of this component since 'Full Width' is related to style. Thoughts on that? PromoCard? Or is FullWidthPromo fine?

Reply:
Ahh that's a good point. I totally agree. 

I'm reluctant to name it anything but FullWidthPromo without discussing with design. Potential names (all a bit rubbish):

- SpecialPromo
- ImagePromo
- BigPromo